### ISSUE TITLE
Linear probing evaluation

### ISSUE THREAD

**Issue by KochPJ** [facebookresearch/dinov2#232]
Hey ho, 

does anyone have the results for the gird search for linear probling evaluation on ImageNet 1k?

- learning rate in {0.0001, 0.0002, 0.0005, 0.001, 0.002, 0.005, 0.01, 0.02, 0.05, 0.1, 0.2, 0.3, 0.5}
- output layers in {1, 4}
- concatenate average-pooled tokens in {yes, no}

i would like to know the constallation used to reproduce the results of the paper. 

best,
Paul

**Comment by qasfb** [comment]
The grid is set by default in the code; it's cheap to execute because there is only a single transformer inference, and multiple linear classifiers are training following the grid, as done in linear.py. You need to use a global batch size of 1024 for this to work properly (as the learning rates are scaled following a scaling rule in the code.

**Comment by KochPJ** [comment]
Allright, so in the code i found a batch size of 126 (https://github.com/facebookresearch/dinov2/blob/main/dinov2/eval/linear.py#L140) which is wrong? So you are using a global batch size of 1024 for your training? And the learning rates are rescaled by lr * (bs* num_worlds) / 256 (https://github.com/facebookresearch/dinov2/blob/main/dinov2/eval/linear.py#L232) ? Where num_worlds is 1 in my case on a single gpu?

**Comment by qasfb** [comment]
Exactly !
